{{cookiecutter.repo_name}}/ci/templates/.travis.yml

```diff
@@ -1,9 +1,12 @@
-language: python
-python: '3.5'
```

Comment:
If you have `python: 3.4` in `.travis.yml` and you run something with python 3.5 (like from tox) it will fail.